{
  "uid" : "5ad361a7b02ad4ea",
  "name" : "test_modify_approve",
  "fullName" : "tests.brownie.token.tests.test_approve#test_modify_approve",
  "historyId" : "955e3f6226713de80a392f10c503148c",
  "time" : {
    "start" : 1641982476906,
    "stop" : 1641982476906,
    "duration" : 0
  },
  "status" : "broken",
  "statusMessage" : "ValueError: Gas estimation failed: 'HTTPConnectionPool(host='10.2.0.2', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f18961e3b80>: Failed to establish a new connection: [Errno 111] Connection refused'))'. This transaction will likely revert. If you wish to broadcast, you must set the gas limit manually.",
  "statusTrace" : "brownie_project = <Project 'TokenProject'>\naccounts = <brownie.network.account.Accounts object at 0x7f3a771bb460>\n\n    @pytest.fixture(scope=\"module\")\n    def token(brownie_project, accounts):\n>       return brownie_project.Token.deploy(\"Test Token\", \"TST\", 18, 1e21, {'from': accounts[0]})\n\ntests/brownie/token/tests/conftest.py:49: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n../../../.local/share/virtualenvs/neon-compatibility-g882AfFF/lib/python3.8/site-packages/brownie/network/contract.py:600: in __call__\n    return tx[\"from\"].deploy(\n../../../.local/share/virtualenvs/neon-compatibility-g882AfFF/lib/python3.8/site-packages/brownie/network/account.py:508: in deploy\n    receipt, exc = self._make_transaction(\n../../../.local/share/virtualenvs/neon-compatibility-g882AfFF/lib/python3.8/site-packages/brownie/network/account.py:725: in _make_transaction\n    raise VirtualMachineError(e) from None\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = VirtualMachineError(ValueError(\"Gas estimation failed: 'HTTPConnectionPool(host='10.2.0.2', port=8899): Max retries ex...ion refused'))'. This transaction will likely revert. If you wish to broadcast, you must set the gas limit manually.\"))\nexc = \"Gas estimation failed: 'HTTPConnectionPool(host='10.2.0.2', port=8899): Max retries exceeded with url: / (Caused by N...ction refused'))'. This transaction will likely revert. If you wish to broadcast, you must set the gas limit manually.\"\n\n    def __init__(self, exc: ValueError) -> None:\n        try:\n            exc = exc.args[0]\n        except Exception:\n            pass\n    \n        if isinstance(exc, dict) and \"message\" in exc:\n            if \"data\" not in exc:\n                raise ValueError(exc[\"message\"]) from None\n    \n            self.message: str = exc[\"message\"].rstrip(\".\")\n    \n            if isinstance(exc[\"data\"], str):\n                # handle parity exceptions - this logic probably is not perfect\n                if \"0x08c379a0\" in exc[\"data\"]:\n                    revert_type, err_msg = [i.strip() for i in exc[\"data\"].split(\"0x08c379a0\", 1)]\n                    err_msg = eth_abi.decode_abi([\"string\"], HexBytes(err_msg))\n                    err_msg = f\"{revert_type} '{err_msg}'\"\n                elif exc[\"data\"].endswith(\"0x\"):\n                    err_msg = exc[\"data\"][:-2].strip()\n                else:\n                    err_msg = exc[\"data\"]\n                raise ValueError(f\"{self.message}: {err_msg}\") from None\n    \n            try:\n                txid, data = next((k, v) for k, v in exc[\"data\"].items() if k.startswith(\"0x\"))\n            except StopIteration:\n                raise ValueError(exc[\"message\"]) from None\n    \n            self.txid: str = txid\n            self.source: str = \"\"\n            self.revert_type: str = data[\"error\"]\n            self.pc: Optional[str] = data.get(\"program_counter\")\n            if self.pc and self.revert_type == \"revert\":\n                self.pc -= 1\n    \n            self.revert_msg: Optional[str] = data.get(\"reason\")\n            self.dev_revert_msg = brownie.project.build._get_dev_revert(self.pc)\n            if self.revert_msg is None and self.revert_type in (\"revert\", \"invalid opcode\"):\n                self.revert_msg = self.dev_revert_msg\n            elif self.revert_msg == \"Failed assertion\":\n                self.revert_msg = self.dev_revert_msg or self.revert_msg\n    \n        else:\n>           raise ValueError(str(exc)) from None\nE           ValueError: Gas estimation failed: 'HTTPConnectionPool(host='10.2.0.2', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f18961e3b80>: Failed to establish a new connection: [Errno 111] Connection refused'))'. This transaction will likely revert. If you wish to broadcast, you must set the gas limit manually.\n\n../../../.local/share/virtualenvs/neon-compatibility-g882AfFF/lib/python3.8/site-packages/brownie/exceptions.py:121: ValueError",
  "flaky" : false,
  "newFailed" : false,
  "newBroken" : false,
  "newPassed" : false,
  "retriesCount" : 0,
  "retriesStatusChange" : false,
  "beforeStages" : [ {
    "name" : "configure_neon_network",
    "time" : {
      "start" : 1641982460018,
      "stop" : 1641982460018,
      "duration" : 0
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  }, {
    "name" : "brownie_project",
    "time" : {
      "start" : 1641982460753,
      "stop" : 1641982475591,
      "duration" : 14838
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  }, {
    "name" : "setup_env",
    "time" : {
      "start" : 1641982475979,
      "stop" : 1641982475979,
      "duration" : 0
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  }, {
    "name" : "accounts",
    "time" : {
      "start" : 1641982475979,
      "stop" : 1641982476248,
      "duration" : 269
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  }, {
    "name" : "token",
    "time" : {
      "start" : 1641982476248,
      "stop" : 1641982476455,
      "duration" : 207
    },
    "status" : "broken",
    "statusMessage" : "ValueError: Gas estimation failed: 'HTTPConnectionPool(host='10.2.0.2', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f18961e3b80>: Failed to establish a new connection: [Errno 111] Connection refused'))'. This transaction will likely revert. If you wish to broadcast, you must set the gas limit manually.\n",
    "statusTrace" : "  File \"/home/runner/.local/share/virtualenvs/neon-compatibility-g882AfFF/lib/python3.8/site-packages/pluggy/_callers.py\", line 39, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/runner/.local/share/virtualenvs/neon-compatibility-g882AfFF/lib/python3.8/site-packages/_pytest/fixtures.py\", line 1126, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/home/runner/.local/share/virtualenvs/neon-compatibility-g882AfFF/lib/python3.8/site-packages/_pytest/fixtures.py\", line 932, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n  File \"/home/runner/work/neon-compatibility/neon-compatibility/tests/brownie/token/tests/conftest.py\", line 49, in token\n    return brownie_project.Token.deploy(\"Test Token\", \"TST\", 18, 1e21, {'from': accounts[0]})\n  File \"/home/runner/.local/share/virtualenvs/neon-compatibility-g882AfFF/lib/python3.8/site-packages/brownie/network/contract.py\", line 600, in __call__\n    return tx[\"from\"].deploy(\n  File \"/home/runner/.local/share/virtualenvs/neon-compatibility-g882AfFF/lib/python3.8/site-packages/brownie/network/account.py\", line 508, in deploy\n    receipt, exc = self._make_transaction(\n  File \"/home/runner/.local/share/virtualenvs/neon-compatibility-g882AfFF/lib/python3.8/site-packages/brownie/network/account.py\", line 725, in _make_transaction\n    raise VirtualMachineError(e) from None\n  File \"/home/runner/.local/share/virtualenvs/neon-compatibility-g882AfFF/lib/python3.8/site-packages/brownie/exceptions.py\", line 121, in __init__\n    raise ValueError(str(exc)) from None\n",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : true,
    "hasContent" : true
  } ],
  "afterStages" : [ {
    "name" : "accounts::0",
    "time" : {
      "start" : 1641982505103,
      "stop" : 1641982505103,
      "duration" : 0
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  } ],
  "labels" : [ {
    "name" : "feature",
    "value" : "Brownie"
  }, {
    "name" : "parentSuite",
    "value" : "tests.brownie.token.tests"
  }, {
    "name" : "suite",
    "value" : "test_approve"
  }, {
    "name" : "host",
    "value" : "fv-az28-122"
  }, {
    "name" : "thread",
    "value" : "3612-MainThread"
  }, {
    "name" : "framework",
    "value" : "pytest"
  }, {
    "name" : "language",
    "value" : "cpython3"
  }, {
    "name" : "package",
    "value" : "tests.brownie.token.tests.test_approve"
  }, {
    "name" : "resultFormat",
    "value" : "allure2"
  } ],
  "parameters" : [ ],
  "links" : [ ],
  "hidden" : false,
  "retry" : false,
  "extra" : {
    "severity" : "normal",
    "retries" : [ ],
    "categories" : [ {
      "name" : "Test defects",
      "matchedStatuses" : [ ],
      "flaky" : false
    } ],
    "history" : {
      "statistic" : {
        "failed" : 0,
        "broken" : 14,
        "skipped" : 0,
        "passed" : 63,
        "unknown" : 0,
        "total" : 77
      },
      "items" : [ {
        "uid" : "f02986cd03d8d09a",
        "reportUrl" : "https://neonlabsorg.github.io/neon-compatibility/460//#testresult/f02986cd03d8d09a",
        "status" : "broken",
        "statusDetails" : "ValueError: Gas estimation failed: 'HTTPConnectionPool(host='10.2.0.2', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f189611ea30>: Failed to establish a new connection: [Errno 111] Connection refused'))'. This transaction will likely revert. If you wish to broadcast, you must set the gas limit manually.",
        "time" : {
          "start" : 1641980265770,
          "stop" : 1641980265770,
          "duration" : 0
        }
      }, {
        "uid" : "1168a3430c65a825",
        "reportUrl" : "https://neonlabsorg.github.io/neon-compatibility/459//#testresult/1168a3430c65a825",
        "status" : "passed",
        "time" : {
          "start" : 1641975781380,
          "stop" : 1641975784806,
          "duration" : 3426
        }
      }, {
        "uid" : "a4d18411f4ecba0a",
        "reportUrl" : "https://neonlabsorg.github.io/neon-compatibility/458//#testresult/a4d18411f4ecba0a",
        "status" : "broken",
        "statusDetails" : "ValueError: Gas estimation failed: 'HTTPConnectionPool(host='10.2.0.2', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f189611b8e0>: Failed to establish a new connection: [Errno 111] Connection refused'))'. This transaction will likely revert. If you wish to broadcast, you must set the gas limit manually.",
        "time" : {
          "start" : 1641974825751,
          "stop" : 1641974825751,
          "duration" : 0
        }
      }, {
        "uid" : "d115b0a594e3c937",
        "reportUrl" : "https://neonlabsorg.github.io/neon-compatibility/457//#testresult/d115b0a594e3c937",
        "status" : "broken",
        "statusDetails" : "ValueError: Gas estimation failed: 'HTTPConnectionPool(host='10.2.0.2', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f1895fcda90>: Failed to establish a new connection: [Errno 111] Connection refused'))'. This transaction will likely revert. If you wish to broadcast, you must set the gas limit manually.",
        "time" : {
          "start" : 1641974204235,
          "stop" : 1641974204235,
          "duration" : 0
        }
      }, {
        "uid" : "9d365d1b2ca970ed",
        "reportUrl" : "https://neonlabsorg.github.io/neon-compatibility/456//#testresult/9d365d1b2ca970ed",
        "status" : "broken",
        "statusDetails" : "requests.exceptions.ReadTimeout: HTTPConnectionPool(host='proxy.night.stand.neontest.xyz', port=80): Read timed out. (read timeout=30)",
        "time" : {
          "start" : 1641957702556,
          "stop" : 1641957702556,
          "duration" : 0
        }
      }, {
        "uid" : "f6fad329ffc1084e",
        "reportUrl" : "https://neonlabsorg.github.io/neon-compatibility/455//#testresult/f6fad329ffc1084e",
        "status" : "broken",
        "statusDetails" : "requests.exceptions.ReadTimeout: HTTPConnectionPool(host='proxy.night.stand.neontest.xyz', port=80): Read timed out. (read timeout=30)",
        "time" : {
          "start" : 1641951899014,
          "stop" : 1641951899014,
          "duration" : 0
        }
      }, {
        "uid" : "4a33a66823895d01",
        "reportUrl" : "https://neonlabsorg.github.io/neon-compatibility/454//#testresult/4a33a66823895d01",
        "status" : "broken",
        "statusDetails" : "requests.exceptions.ReadTimeout: HTTPConnectionPool(host='proxy.night.stand.neontest.xyz', port=80): Read timed out. (read timeout=30)",
        "time" : {
          "start" : 1641950377450,
          "stop" : 1641950377450,
          "duration" : 0
        }
      }, {
        "uid" : "681b19953d9256a1",
        "reportUrl" : "https://neonlabsorg.github.io/neon-compatibility/453//#testresult/681b19953d9256a1",
        "status" : "passed",
        "time" : {
          "start" : 1641947684990,
          "stop" : 1641947687812,
          "duration" : 2822
        }
      }, {
        "uid" : "cf7ad59e6ceae665",
        "reportUrl" : "https://neonlabsorg.github.io/neon-compatibility/452//#testresult/cf7ad59e6ceae665",
        "status" : "passed",
        "time" : {
          "start" : 1641870928562,
          "stop" : 1641870931466,
          "duration" : 2904
        }
      }, {
        "uid" : "b15909f2c250da52",
        "reportUrl" : "https://neonlabsorg.github.io/neon-compatibility/451//#testresult/b15909f2c250da52",
        "status" : "passed",
        "time" : {
          "start" : 1641861316196,
          "stop" : 1641861319938,
          "duration" : 3742
        }
      }, {
        "uid" : "d33a3d13ee44e7d",
        "reportUrl" : "https://neonlabsorg.github.io/neon-compatibility/450//#testresult/d33a3d13ee44e7d",
        "status" : "passed",
        "time" : {
          "start" : 1641784482028,
          "stop" : 1641784485770,
          "duration" : 3742
        }
      }, {
        "uid" : "9f80facb059fa6c1",
        "reportUrl" : "https://neonlabsorg.github.io/neon-compatibility/449//#testresult/9f80facb059fa6c1",
        "status" : "passed",
        "time" : {
          "start" : 1641774822039,
          "stop" : 1641774825319,
          "duration" : 3280
        }
      }, {
        "uid" : "f2c467659a78d86b",
        "reportUrl" : "https://neonlabsorg.github.io/neon-compatibility/448//#testresult/f2c467659a78d86b",
        "status" : "passed",
        "time" : {
          "start" : 1641698103057,
          "stop" : 1641698106087,
          "duration" : 3030
        }
      }, {
        "uid" : "46bc455b23e02b59",
        "reportUrl" : "https://neonlabsorg.github.io/neon-compatibility/447//#testresult/46bc455b23e02b59",
        "status" : "passed",
        "time" : {
          "start" : 1641688595102,
          "stop" : 1641688598805,
          "duration" : 3703
        }
      }, {
        "uid" : "7bdbc8ece19793cf",
        "reportUrl" : "https://neonlabsorg.github.io/neon-compatibility/446//#testresult/7bdbc8ece19793cf",
        "status" : "broken",
        "statusDetails" : "ValueError: Verifying nonce before send transaction: Error processing Instruction 1: invalid program argument",
        "time" : {
          "start" : 1641611739179,
          "stop" : 1641611770249,
          "duration" : 31070
        }
      }, {
        "uid" : "2c6b490e2dbc9a5c",
        "reportUrl" : "https://neonlabsorg.github.io/neon-compatibility/445//#testresult/2c6b490e2dbc9a5c",
        "status" : "passed",
        "time" : {
          "start" : 1641602011721,
          "stop" : 1641602015490,
          "duration" : 3769
        }
      }, {
        "uid" : "46100c0a4c653634",
        "reportUrl" : "https://neonlabsorg.github.io/neon-compatibility/441//#testresult/46100c0a4c653634",
        "status" : "passed",
        "time" : {
          "start" : 1641525431034,
          "stop" : 1641525435376,
          "duration" : 4342
        }
      }, {
        "uid" : "6a0f351461b5952",
        "reportUrl" : "https://neonlabsorg.github.io/neon-compatibility/440//#testresult/6a0f351461b5952",
        "status" : "passed",
        "time" : {
          "start" : 1641515694729,
          "stop" : 1641515697813,
          "duration" : 3084
        }
      }, {
        "uid" : "ab13ba9196cea6de",
        "reportUrl" : "https://neonlabsorg.github.io/neon-compatibility/434//#testresult/ab13ba9196cea6de",
        "status" : "passed",
        "time" : {
          "start" : 1641439019972,
          "stop" : 1641439024092,
          "duration" : 4120
        }
      }, {
        "uid" : "61f10d51876f701f",
        "reportUrl" : "https://neonlabsorg.github.io/neon-compatibility/432//#testresult/61f10d51876f701f",
        "status" : "passed",
        "time" : {
          "start" : 1641429363909,
          "stop" : 1641429368238,
          "duration" : 4329
        }
      } ]
    },
    "tags" : [ ]
  },
  "source" : "5ad361a7b02ad4ea.json",
  "parameterValues" : [ ]
}